resources/scripts/components/server/features/index.ts

```diff
@@ -7,5 +7,7 @@ import { lazy } from 'react';
  * on the feature and the egg).
  */
 const EulaModalFeature = lazy(() => import(/* webpackChunkName: "feature.eula" */'@feature/eula/EulaModalFeature'));
+const JavaVersionModalFeature = lazy(() => import(/* webpackChunkName: "feature.javaversion" */'@feature/JavaVersionModalFeature'));
```

Comment:
Replace `"feature.javaversion"` with `"feature.javaVersion"`

Reply:
Fixed in next commit